### ISSUE TITLE
[a11y][iOS] Screen reader should outline entire card on CollectionView and CarouselView

### ISSUE THREAD

**Issue by rachelkang** [rachelkang/recipeSearch#6]
On iOS, the screen reader outline/border only outlines the recipe name on CollectionView and CarouselView cards.

On Android, however, the screen reader outline/border outlines the entire card (including the image) as a single unit and reads out the encapsulated label. This behavior should be seen on iOS as well.
